hikari/impl/entity_factory.py

```diff
@@ -1718,9 +1721,10 @@ def deserialize_command(
         return commands.Command(
             app=self._app,
             id=snowflakes.Snowflake(payload["id"]),
+            type=commands.CommandType(payload.get("type", commands.CommandType.CHAT_INPUT)),
```

Comment:
is this default actually necessary or does discord only document the type field as being optional because its not required in the command objects you send to them while always being included in the command objects we get back from them